src/actions/__stories__/Button.stories.tsx

```diff
@@ -0,0 +1,164 @@
+import React from "react"
+
+import Button from "../Button"
+
+import Icon from "../../icons/Icon"
+import { faHeart } from "@fortawesome/free-solid-svg-icons"
+
+import MDXDocs from "./Button.docs.mdx"
+
+export default {
+  title: "Actions/Button",
+  component: Button,
+  parameters: {
+    docs: {
+      page: MDXDocs,
+    },
+  },
+}
+
+export const defaultButton = () => <Button onClick={() => alert("Clicked!")}>Default Button</Button>
+
+export const buttonVariants = () => (
+  <>
+    <div>
```

Comment:
Could you add headers to each of these sections? Looking at the storybook you can't easily see what size each row is referencing.